core/src/test/scala/com/mesosphere/usi/core/logic/MesosEventsLogicTest.scala

```diff
@@ -0,0 +1,60 @@
+package com.mesosphere.usi.core.logic
+
+import com.mesosphere.mesos.client.MesosCalls
+import com.mesosphere.usi.core.helpers.FrameworkMock
+import com.mesosphere.usi.core.helpers.MesosMock
+import com.mesosphere.usi.core.matching.ScalarResourceRequirement
+import com.mesosphere.usi.core.models.PodId
+import com.mesosphere.usi.core.models.ResourceType
+import com.mesosphere.utils.UnitTest
+
+class MesosEventsLogicTest extends UnitTest {
+
+  private val mesosEventLogic = new MesosEventsLogic(new MesosCalls(FrameworkMock.mockFrameworkId))
+
+  "MesosEventsLogic" should {
+    "decline an offer when PodSpec's are empty" in {
```

Comment:
Podspecs... maybe we can do `decline an offer when there is no podspec to launch`? I feel like that's less implementation specific name :)